recipes/bamhash/1.1/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+export LDFLAGS=-L$PREFIX/lib
+export CXXFLAGS=-I$PREFIX/include
+
+make all
+
+mv bamhash_checksum_bam $PREFIX/bin
+mv bamhash_checksum_fastq $PREFIX/bin
```

Comment:
Odd, it works for me in the bioconda docker container. I'm not specifying the python3 build environment, though, so perhaps that's the difference (no clue how that could happen though).